core/state_transition.go

```diff
@@ -278,6 +285,12 @@ func (st *stateTransition) buyGas() error {
 		}
 	}
 	balanceCheck.Add(balanceCheck, st.msg.Value)
+	if l1Cost != nil {
```

Comment:
Why does this line exist? I am confused, it seems unrelated to operator fee

Reply:
we already add `l1Cost` above to `balanceCheck`

Reply:
It maybe a bad merge

Reply:
Fixed